test-result-summary-client/src/PerfCompare/lib/JenkinsRunJSON.js

```diff
@@ -13,11 +11,10 @@ export default class JenkinsRunJSON {
         let curVariantObject = null;
         let curVariantObjectId = null;
 
-        if (this.runJSON.testInfo[0].tests !== undefined) {
```

Comment:
Earlier, we were mainly using this function to calculate `mean` and `CI`, but now we already have that data since we do all the calculation once a build is done. So we need to verify if we really need some of the code in this file.